### ISSUE TITLE
Improve GitHub microsite accessibility

### ISSUE THREAD

**Issue by BinaryMuse** [atom/atom.io#119]
From https://github.com/atom/github/issues/832:

> I notice the [githb-atom website](https://github.atom.io/) is not accessible for all users.
> 
> the navbar inside the `main.features` have a list of spans that's is when you click on the video above change but as you know the span element is not focusable so you should use `<button>` instead of `<span>` to make it accessible for keyboard users.
> 
> also, the website a11y need to more work I notice also the icons do not have `aria-hidden=true` or `aria-label` whatever you want to the screen reader user read.

/cc @Muhnad who opened the original issue
/cc @jasonrudolph @simurai

**Comment by jasonrudolph** [comment]
@Muhnad: Thank you for bringing this to our attention. We've updated https://github.atom.io to address these issues. If you continue to notice any accessibility issues on https://github.atom.io, please let us know.
